lib/reporter/results.js

```diff
@@ -286,7 +290,10 @@ module.exports = class Results {
       retries: testcase.retriesCount,
       skipped: 0,
       tests: 0,
-      status: Results.TEST_PASS
+      status: Results.TEST_PASS,
+      startTimestamp: new Date().getTime(),
+      endTimestamp: new Date().getTime(),
```

Comment:
Shouldn't `endTimestamp` be caluclated on completion of test section.

Reply:
```suggestion
```

Reply:
I don't think either start or end timestamp is required here.